src/applications/differential/storage/DifferentialRevision.php

```diff
@@ -1127,7 +1127,14 @@ public function getFieldValuesForConduit() {
       'color.ansi' => $status->getANSIColor(),
     );
 
+    $stack_graph = id(new DifferentialRevisionGraph())
+        ->setSeedPHID($this->getPHID())
+        ->setLoadEntireGraph(true)
+        ->loadGraph();
+
     return array(
+      'depends_on' => $stack_graph->getEdges(DifferentialRevisionDependsOnRevisionEdgeType::EDGECONST,),
+      'dependents' => $stack_graph->getEdges(DifferentialRevisionDependedOnByRevisionEdgeType::EDGECONST),
```

Comment:
```suggestion
```

This is no longer needed or used.

Reply:
Does it cost us any performance to include the entire stack in both directions? Seems like we've already calculated it, could be useful to have in the future.

Reply:
I don't think there's additional DB queries for this, since `$stack_graph` is already loaded.

Lando's terminology (at least in all the revision worker implementation) is `dependents`, and I think it's a little more standard than `depended_on_by`. I'm not super picky here, but I am inclined to not include it since it is not used/useful for the purposes of calculating edges, and if we need it in the future we can just add this line again.

It seems that the standard way of representing DAGs is in a dictionary with nodes as keys and their predecessor nodes as a list of values, not the other way around. Therefore I don't think we'll necessary need this in the future.

Reply:
As a bike shed, if we're going with non-Phab terminology we should consider `descendants` and `ancestors` for those revisions. That's the `evolve` convention and the clearest terms I've seen used to describe parent/child relationships. I'm not super picky either so I'd rather not split hairs but worth throwing out there as those have the advantage of not being based on the same root word `depend` and being confused about which term refers to which topological direction.

Makes sense, leaving them out should be fine and it seems those two edge types are often coupled together in the code base - shouldn't be hard to find them again if we needed to add the other direction to the response.

Reply:
> As a bike shed, if we're going with non-Phab terminology we should consider `descendants` and `ancestors` for those revisions. That's the `evolve` convention and the clearest terms I've seen used to describe parent/child relationships. I'm not super picky either so I'd rather not split hairs but worth throwing out there as those have the advantage of not being based on the same root word `depend` and being confused about which term refers to which topological direction.

More bikeshedding here, I dislike terminology that implies an objective hierarchical relationship between nodes (e.g. "parent" and "child", or "predecessor" and "ancestor"). In the context of revisions, there is nothing but a directional dependency when it comes to the relationship (i.e. predecessor/successor might make a little more sense if we want to move away from the language of dependency). I would opt for predecessor/successor over descendant/ancestor (which I intentionally try to avoid).

> Makes sense, leaving them out should be fine and it seems those two edge types are often coupled together in the code base - shouldn't be hard to find them again if we needed to add the other direction to the response.

I think given that we are only leaving one field here ("depends_on") and that this terminology is not necessarily problematic and it matches with the current Phabricator and Lando terminology, this should be left as-is. Thoughts?